### PR TITLE
[0.13] release notes: Mention new relevance of non-mining nodes on network policy; and misc fixes

### PR DESCRIPTION
**Pull request by luke-jr**

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -130,25 +159,26 @@ cases it also reduces propagation delay. It is automatically enabled between
...
-You can't disable HD key generation once you have created a HD wallet.
+Deterministic key generation for new wallets can be disabled by `-usehd=0`.
+Keep in mind that this flag only has affect on newly created wallets. You
+can't disable deterministic key generation once you have created a
+deterministic wallet.
 
 There is no distinction between internal (change) and external keys.
 
-HD wallets are incompatible with older versions of Bitcoin Core.
+Deterministic wallets are incompatible with older versions of Bitcoin Core.
```
Comment (rebroad) [inline doc/release-notes.md]: "HD" is no longer the term used?

Comment (luke-jr) [inline doc/release-notes.md]: "HD" is the term for hierarchical deterministic wallets. I am making the observation that Core 0.13's is not really hierarchical. TBD if anyone agrees of I should back this change out.

**Comment by jonasschnelli** [comment]
I think it is confusing if we would use a different term then HD (hierarchical deterministic). We are using HD. The seed is not on the same hierarchical level then the derived keys. We are deriving `m/0'/0'/k'` which is kind of the proposed path after BIP32.

**Comment by luke-jr** [comment]
Removed HD wallet changes.

File: doc/release-notes.md [hunk 2]
```diff
@@ -38,6 +38,35 @@ report issues about Windows XP to the issue tracker.
 Notable changes
 ===============
 
+Non-mining nodes may influence miner policy
+-------------------------------------------
+
+As a side-effect of Compact Blocks support (see below), ordinary non-mining
+nodes will download and upload blocks faster if those blocks were produce by
```
Comment (sipa) [inline doc/release-notes.md]: produced

**Comment by sipa** [comment]
ACK the sigops change section.

Regarding the relay influence section:
* I don't think the CB effect is very significant... miners can and will peer with each other, even ignoring things like faster relay network protocols.
* When ignoring non-random-public-P2P-network relay between miners, full nodes have always had influence through policy decisions through the signature cache.
* The strongest effect non-mining full nodes have is through economic majority agreeing or disagreeing with consensus changes, which is independent from policy.

I do think it is relevant to mention the change here, but maybe we can just turn it into a paragraph in the compact blocks section?

**Comment by luke-jr** [comment]
@sipa How's this?

**Comment by sipa** [comment]
ACK

**Comment by MarcoFalke** [comment]
Needs merge conflict solved, if still relevant.

**Comment by sipa** [comment]
Rebased version: https://github.com/sipa/bitcoin/commits/relnotes-luke

**Comment by luke-jr** [comment]
Thanks, pushed that here.

**Pull request merged**